files/ja/web/api/fetch_api/basic_concepts/index.html

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
after rebase of the PR had added some un-relative files. 

I feel that we may need to separate the PR into each locale, making it to be easier to review and merge separately.

And to ensure it's not been blocked longly and needs of continence rebase, reduce the risk of broken during each rebase.

Reply:
will update shortly